tests/integration/test_enrollment/test_agent_auth_enrollment.py

```diff
@@ -0,0 +1,74 @@
+'''
```

Comment:
The module is not pep8 compliant. 
![4](https://user-images.githubusercontent.com/13010397/131728775-f7300f87-8e13-4f27-beea-7ddb0564e654.png)

Reply:
Fixed